### PR TITLE
Forms Automated Test - LPS-133811 | master

### PR DESCRIPTION
**Pull request by ewertonluna**

### CODE REVIEW

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 1]
```diff
@@ -98,6 +98,11 @@
 	<td>//div[@class='ddm-field' and contains(@data-field-name,'${key_fieldName}')]//label[text()='${key_fieldLabel}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>OBJECT_FIELD_BUTTON_DROPDOWN</td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path:102]: @ewertonluna, please rename to `FIELD_DROPDOWN_ITEM`

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormRules.path [hunk 2]
```diff
@@ -85,6 +85,11 @@
     <td></td>
 </tr>
 <!--RULE_BUILDER_AUTOFILL-->
+<tr>
+	<td>RULE_BUILDER_AUTOFILL_FIELD_BY_INDEX</td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormRules.path:89]: @ewertonluna, is it really necessary on this pull? Can you please clarify?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormRules.path (hunk 2 above)
Comment (ewertonluna) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormRules.path]: Hi @natocesarrego! It really wasn't necessary, so I removed it.

**Comment by ewertonluna** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**